src/MAUI/Maui.Samples/Samples/Layers/AddVectorTiledLayerFromCustomStyle/AddVectorTiledLayerFromCustomStyle.xaml.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Copyright 2022 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Portal;
+using Esri.ArcGISRuntime.Tasks.Offline;
+
+namespace ArcGIS.Samples.AddVectorTiledLayerFromCustomStyle
+{
+    [ArcGIS.Samples.Shared.Attributes.Sample(
+        name: "Add vector tiled layer from custom style",
+        category: "Layers",
+        description: "Load ArcGIS vector tiled layers using custom styles.",
+        instructions: "Pan and zoom to explore the vector tile basemap.",
+        tags: new[] { "tiles", "vector", "vector basemap", "vector tiled layer", "vector tiles" })]
+    [ArcGIS.Samples.Shared.Attributes.OfflineData()]
```

Comment:
Should I add "e01262ef2a4f4d91897d9bbd3a9b1075" and "ce8a34e5d4ca4fa193a097511daa8855" here?

Reply:
Not needed. The data is not downloaded through the viewer.